src/artm/core/dictionary_operations.cc

```diff
@@ -94,6 +94,10 @@ void DictionaryOperations::Export(const ExportDictionaryArgs& args, const Dictio
   }
 
   std::string str = token_dict_data.SerializeAsString();
+  if (str.size() >= 2147483647ULL)
+    BOOST_THROW_EXCEPTION(InvalidOperation("Dictionary " +
+      args.dictionary_name() + " is too large to export"));
```

Comment:
minor: for export

Reply:
Either "to export" or "for exporting". Imho "to export" is more english.